cli-plugins/manager/manager_windows.go

```diff
@@ -0,0 +1,11 @@
+package manager
+
+import (
+	"os"
+	"path/filepath"
+)
+
+var defaultPluginDirs = []string{
+	filepath.Join(os.Getenv("USERPROFILE"), `.docker\cli-plugins`),
+	`C:\ProgramData\Docker\cli-plugins`,
```

Comment:
Not really sure what to do about this, given that Windows doesn't (appear to) have an equivalent "global" default akin to `/usr/foo` and `/usr/local/foo`. 

The engine has a bunch of what appear to be [hardcoded values](https://github.com/moby/moby/blob/master/container/container_windows.go#L15...L17) in the same path (if those are configurable my `git grep` is not finding it). 

It is possible to augment this set of paths via `~/.docker/config.json` though -- which I think is sufficient if things are installed in other non-default paths.

Reply:
I've honestly no idea, there were comments on the design which lead me to this particular set, perhaps @simonferquel you could advise?

Like with the other comment, there is a user specific one too, it's in `defaultUserPluginDir` here which is (now) `defaultSystemPluginDirs` but it isn't here because it is the same on both Unix and Windows (since it uses `github.com/docker/cli/cli/config` which already does whatever abstraction is needed)

Reply:
os.GetEnv("ProgramData")  should give you the shared program data dir (common to all users) with default value being c:\ProgramData (can change if you install Windows on a different drive than C)